legend-engine-xt-relationalStore-grammar/pom.xml

```diff
@@ -216,11 +216,16 @@
             <artifactId>legend-engine-xt-json-model</artifactId>
             <scope>test</scope>
         </dependency>
-<!--        <dependency>-->
-<!--            <groupId>platform.alloy</groupId>-->
-<!--            <artifactId>legend-engine-language-pure-dsl-service</artifactId>-->
-<!--            <scope>test</scope>-->
-<!--        </dependency>-->
+        <dependency>
```

Comment:
This seems like something which can give rise to a cyclic dependency structure. I had to add such dependencies for tests myself, maybe we should have a module for these tests? What do you think?

Reply:
The dependencies are scoped for only tests. If they do lead to cyclic dependency we can move them out to its own module but for now I think its fine.